README.md

```diff
@@ -1,4 +1,440 @@
 # NervesTeamGame
 
-Server side game engine for NervesTeam
+NervesTeam Game Logic
 
+This application provides the game logic for playing NervesTeam. It handles
+messages from game clients and provides instructions for what those clients
+should show next on their screens. All game interactions are through
+function calls. Other modules glue this up to Phoenix Channels (or any
+other transport). The API uses JSON strings to pass messages that are documented below.
```

Comment:
I was curious about your choice of saying its using `JSON` strings. This application only delivers maps. They are converted to JSON to send over the wire, if you were to be using channels. Should we change all these examples to illustrate maps?